### DISCUSSION TITLE
Should content pertaining to `<blink>` be deleted?

### DISCUSSION THREAD

**Discussion by MrBrain295** [mdn/mdn-community#296] [Content] [answered]
`<blink>` is no longer supported by any major browser. Should it's documentation be removed? If so I would be happy to work on it.

**Comment by teoli2003** [comment, marked as answer]
On one side, it surely matches our deletion rules. On another side, there is a Web History behind it; I think we should keep the page and update it to tell this story (the bet in a Mountain View pub), and link to a recommendation not to do blinking text (for a11y).

**Comment by bsmth** [comment]
I'm going to close this as resolved for now, but there's been some movement to add Glossary entries for notable deprecated features:

- [x] https://github.com/mdn/content/pull/35975

Related discussions:

- [Should we write about old features that any current browsers does (or doesn't) implement?](https://github.com/orgs/mdn/discussions/589)
- ["to be avoided" and deprecated features #438](https://github.com/orgs/mdn/discussions/589)
- [ Is there information on deprecated or obsolete elements and properties in MDN? #344 ](https://github.com/orgs/mdn/discussions/344)

Thank you!
